home_task_queries.sql

```diff
@@ -0,0 +1,133 @@
+/*
+ Завдання на SQL до лекції 02.
+ */
+
+
+/*
+1.
+Вивести кількість фільмів в кожній категорії.
+Результат відсортувати за спаданням.
+*/
+with filmCountByFilmCategory(category_id, film_count) as
+         (select category_id,
+                 count(*) as film_count
+          from film_category
+          group by category_id)
+
+select c.name, s.film_count
+from filmCountByFilmCategory as s
+         left join category as c on s.category_id = c.category_id
+order by s.film_count desc
+
+
+
+
+/*
+2.
+Вивести 10 акторів, чиї фільми брали на прокат найбільше.
+Результат відсортувати за спаданням.
+*/
+with filmRentCount (film_id, rent_count) as
```

Comment:
The same here. WITH is good when it simplifies something, but does not make it more complicated.